services/utils/utils_test.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	"testing"
+
+	mevt "maunium.net/go/mautrix/event"
+)
+
+func TestHTMLStrip(t *testing.T) {
+	msg := `before <hello a="b"><inside />during</hello> after`
+	stripped := StrippedHTMLMessage(mevt.MsgNotice, msg)
+	if stripped.MsgType != mevt.MsgNotice {
+		t.Fatalf("Expected MsgType %v, got %v", mevt.MsgNotice, stripped.MsgType)
+	}
+	expected := "before during after"
```

Comment:
It would be worth having some encoded chars in there to make sure `UnescapeString` is working correctly. E.g `&lt;`

Reply:
Good point, added